backend/src/libs/interfaces/entity.interface.ts

```diff
@@ -0,0 +1,6 @@
+interface Entity {
```

Comment:
if interfaces without any prefixes will be used, why use them then at all? what is better than types?

Reply:
They are used for implementation

Reply:
I can implement type 🤔

<img width="716" alt="image" src="https://github.com/BinaryStudioAcademy/bsa-2023-serenity/assets/42296182/0f2700bd-e88c-4509-9c98-b5f4241243d7">

> how does prefix impact interface usage?

No, it doesn't. But right now in the application, there is no difference between interface and type. Why do we need to use interfaces then?

Reply:
Ok, then it's more semantically correct to use interface because its goal is to define a contract and also it restricts you to a plain object, while type can have other options that won't work with implements. So for me, it looks like type defines either data type or structure, while interface definine a contract that can be used only for implementation

Reply:
Okay, `import { User, Admin, Doctor, Man }`, in this case, where is the interface? I do not know 🤔

Reply:
that's a good point, will change them to types

Reply:
Changed this